pipeline/transformer/transformer_calm/src/main/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformer.scala

```diff
@@ -85,6 +85,19 @@ object CalmTransformer
     }
 
   def shouldSuppress(record: CalmRecord): Boolean =
+    transmissionIsNo(record) match {
+      case true  => true
+      case false =>
+        // Records prefixed with AMSG (Archives and Manuscripts Resource Guides)
+        // are not actual archives but instead guides for researches, so we
+        // suppress them here.
+        record
+          .get("RefNo")
+          .map(_.startsWith("AMSG"))
+          .getOrElse(false)
```

Comment:
Trivia: you can express this boolean in one statement rather than using `getOrElse`
```suggestion
          .exists(_.startsWith("AMSG"))
```